ui/src/utils/mod.rs

```diff
@@ -6,7 +6,7 @@ use crate::state::Theme;
 
 pub mod format_timestamp;
 pub mod language;
-pub mod notifications;
+//pub mod notifications;
```

Comment:
@InfamousVague  will we be using our custom notifications from now on? if so then the code relying on `rust-notify` can be removed.

Reply:
We will be using external notifications, the internal ones are for errors and things like "copied to clipboard successfully"